spec/features/comment_pages/comment_feed_spec.rb

```diff
@@ -1,42 +1,52 @@
-# Define the following let variables before using these examples:
-#
-#   create_comments : a block which creates the comments AND IS CALLED
-#                     BEFORE THE PAGE LOADS
-#   commentable: the model which the 'show' page is about
-shared_examples 'a page with a comments feed' do |commentable_factory|
+require 'rails_helper'
+
+describe 'Comment feed', js: true do
+  before { login_to_project_as_user }
 
   include CommentMacros
 
-  let(:create_comments) do
-    @comments = [
-      create(:comment, commentable: commentable, user: @logged_in_as),
-      create(:comment, commentable: commentable)
+  let!(:issue) { create(:issue, node: current_project.issue_library) }
+
+  let!(:comments) do
+    [
+      create(:comment, commentable: issue, user: @logged_in_as),
+      create(:comment, commentable: issue)
     ]
-    other_instance = create(commentable_factory || commentable.class.to_s.underscore)
-    @other_comment = create(:comment, commentable: other_instance)
   end
 
-  it 'lists them in the content feed', js: true do
+  before do
+    visit project_issue_path(current_project, issue)
+
     # Wait for ajax
     find('[data-behavior~=fetch-comments] .comment-feed')
+  end
 
-    within comment_feed do
-      should have_comment(@comments[0])
-      should have_comment(@comments[1])
-      should_not have_comment(@other_comment)
+  describe 'list comments' do
```

Comment:
Can we consolidate all the `it` blocks into one and check everything in one go?

Reply:
Done!